### DISCUSSION TITLE
Teleport with conditional elements (listeners not bound?)

### DISCUSSION THREAD

**Discussion by itsgrimace** [livewire/livewire#9067] [7. Bugs (archived)] [answered]
### Livewire version

v3.5.16

### Laravel version

v11.34.2

### Which PHP version are you using?

PHP 8.3

### What is the problem?

When using the @teleport directive if this section has some conditional items (eg @if or @foreach) event listeners are not being bound to any items added after the initial render event. In short wire:click and x-on:click directives are not being triggered (possibly others).

In the provided example the button will always work, the event on item 1 will always work but events for any item 2+ and the conditional item will not work.

This problem has become apparent on upgrade to v3.5.16. The behaviour is not present on v3.5.12.

### Code snippets

This example is the same as the provided Wirebox
Teleport.php
```php
<?php

namespace App\Livewire;

use Livewire\Component;

class Teleport extends Component
{
    public $items = [1];
    public $counter = 1;
    
    public function increment(){
        $this->counter++;
        $this->items[$this->counter] = $this->counter;
    }
    
    public function render()
    {
        return view('livewire.teleport');
    }
}
```
teleport.blade.php
```blade
<div>
    <button class="bg-white border-solid rounded m-5 p-5" wire:click="increment">Add Some Item</button>
    <p class="bg-white">{{$counter}}</p>
    @teleport('body')
    <div>
        <div>
        @foreach($items as $item)
            <div class="m-5 bg-white" wire:click="increment" wire:key="item{{$item}}"> item {{$item}}</div>
        @endforeach
        </div>
        <div>
        @if($counter > 1)
            <div class="m-5 bg-white" x-on:click="$wire.increment"> Conditional item</div>
        @endif
        </div>
    </div>
    @endteleport
</div>
```
Include in view as `@livewire('teleport')`

### Wirebox URL

https://wirebox.app/b/47nge

### How do you expect it to work?

I would expect that livewire events and alpine events are processed as with previous versions

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by danie-ramdhani** [comment]
Yeah, `x-teleport` is broken after alpine v.3.14.4, see https://github.com/alpinejs/alpine/discussions/4467, https://github.com/alpinejs/alpine/discussions/4461.

**Comment by joshhanley** [comment]
I've replied on https://github.com/alpinejs/alpine/discussions/4467#discussioncomment-11479445 about this issue.

Copied message below

> I had a look this with Caleb on our call this morning and we think the teleport `clone._x_ignore = true ` was something for Livewire. But we're not 100% on that. Caleb said to just try removing it and see if both test suites pass.
> 
> So I've submitted an Alpine PR #4469 with it removed and a Livewire PR https://github.com/livewire/livewire/pull/9072 with the Alpine fix and both test suites are passing.

**Comment by joshhanley** [comment, marked as answer]
@itsgrimace this has been fixed and released in v3.5.17
